### PR TITLE
TF-739, TF-737 : Add bullets and align composer tablet

### PR DESCRIPTION
**Pull request by ManhNTX**
https://user-images.githubusercontent.com/107173849/181574299-be05a6aa-dfd6-47e0-8dab-75d2bf5e28db.mov

https://user-images.githubusercontent.com/107173849/181677434-515ebe45-6d79-4bab-9cf5-6a5c7cccc6e8.mov

https://user-images.githubusercontent.com/107173849/181678607-d5f453fc-998d-4c83-9dbb-2a63342645ab.mov

### CODE REVIEW

File: lib/features/composer/presentation/controller/rich_text_mobile_tablet_controller.dart [hunk 1]
```diff
@@ -24,6 +27,15 @@ class RichTextMobileTabletController extends BaseRichTextController {
 
   bool get isMenuHeaderStyleOpen => menuHeaderStyleStatus.value == DropdownMenuFontStatus.open;
 
+  final selectedParagraph = ParagraphType.alignLeft.obs;
+  final selectedOrderList = OrderListType.bulletedList.obs;
+
+  final focusMenuOrderList = RxBool(false);
```
Comment (dab246) [inline lib/features/composer/presentation/controller/rich_text_mobile_tablet_controller.dart:33]: I don't see where you change this value?

File: lib/features/composer/presentation/controller/rich_text_mobile_tablet_controller.dart [hunk 2]
```diff
@@ -24,6 +27,15 @@ class RichTextMobileTabletController extends BaseRichTextController {
...
+  final focusMenuParagraph = RxBool(false);
```
Comment (dab246) [inline lib/features/composer/presentation/controller/rich_text_mobile_tablet_controller.dart:34]: same

**Review by dab246 — COMMENTED** [review]
- You should not format the code, it will be difficult to review
- If possible, can you show me a demo with nested bullets?
Ex:
```html
1.
2.
    1.
    2.
3.
   *
   *
   *
```

**Comment by ManhNTX** [comment]
> * You should not format the code, it will be difficult to review
> * If possible, can you show me a demo with nested bullets?
>   Ex:
> 
> ```
> 1.
> 2.
>     1.
>     2.
> 3.
>    *
>    *
>    *
> ```

I do it. Please check again

**Review by dab246 — APPROVED** [review]
/lgtm

**Review by hoangdat — APPROVED** [review]

**Review by hoangdat — APPROVED** [review]

**Pull request merged**